.github/workflows/linter.yml

```diff
@@ -35,6 +35,8 @@ jobs:
           MULTI_STATUS: false
           VALIDATE_ALL_CODEBASE: false
           VALIDATE_GITHUB_ACTIONS: true
+          LINTER_RULES_PATH: .github/linters
```

Comment:
`.github/linters` is the default, so you can remove this, unless you want to keep it for documentation purposes.